packages/react/src/components/Loader/index.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Loader from './';
+import axe from '../../axe';
+
+test('renders a Loader component', () => {
+  render(<Loader data-testid="loader" />);
+  expect(screen.getByTestId('loader')).toBeInTheDocument();
+});
+
+test('handles classNames properly', () => {
+  render(<Loader data-testid="loader" className="baz" />);
+  expect(screen.getByTestId('loader')).toHaveClass('Loader', 'baz');
+});
+
+test('sets aria-hidden if no label is provided', () => {
+  render(<Loader data-testid="loader" />);
+  expect(screen.getByTestId('loader')).toHaveAttribute('aria-hidden', 'true');
+});
+
+test('does not set aria-hidden if a label is provided', () => {
+  render(<Loader data-testid="loader" label="hi" />);
+  expect(screen.getByTestId('loader')).not.toHaveAttribute(
+    'aria-hidden',
+    'false'
+  );
+});
+
+test('sets expected role attributes given an aria-label', () => {
+  render(<Loader data-testid="loader" label="bananas" />);
+  expect(screen.getByTestId('loader')).toHaveAttribute('role', 'alert');
+});
+
+test('supports ref={React.createRef()}', () => {
```

Comment:
Small nit:

```suggestion
test('supports ref prop', () => {
```

Reply:
Looks like this wasn't updated? This was just to keep the naming consistent with other tests. Just checking.